### PR TITLE
TV Rename Website 2.6 

### PR DESCRIPTION
**Pull request by SirSparkles**
Ready for release?

### CODE REVIEW

File: _manual/features/finding.md [hunk 1]
```diff
@@ -1,56 +1,56 @@
...
+
+### Types
+
+There are 6 places it could search:
+
+* _File System_ – If found it will remove the episode from the missing list and move the file into the correct place.
+
+* _RSS Feed for torrent links_, _JSON Webpage for torrent links_ – If found then it will download the torrent file. (via uTorrent or qBittorrent) See https://beta.tvrename.com/manual/options/#the-rss-json-tab for more information about configuration.
+
+* _SABnzbd_, _uTorrent_, _qBittorrent_ – If found it will mark it as downloading so you know not to try and find the item again. See https://beta.tvrename.com/manual/options/#the-torrents--nzb-tab for more informaiton.
```
Comment (SirSparkles) [inline _manual/features/finding.md]: Link needs to be setup properly

File: _manual/features/finding.md [hunk 2]
```diff
@@ -1,56 +1,56 @@
...
+
+Once a scan is complete, TV Rename will identify files to move/copy/rename and additional files to download/create. For episodes that are still missing, TV Rename will try and find them for you. 
+
+### Types
+
+There are 6 places it could search:
+
+* _File System_ – If found it will remove the episode from the missing list and move the file into the correct place.
+
+* _RSS Feed for torrent links_, _JSON Webpage for torrent links_ – If found then it will download the torrent file. (via uTorrent or qBittorrent) See https://beta.tvrename.com/manual/options/#the-rss-json-tab for more information about configuration.
```
Comment (SirSparkles) [inline _manual/features/finding.md]: Link needs to be setup properly

**Review by Andy-ABTec — APPROVED** [review]
I'm happy if you are...

File: _manual/features/finding.md (hunk 1 above)
Comment (Andy-ABTec) [reply _manual/features/finding.md]: I'll go fix it now...

File: _manual/features/finding.md (hunk 2 above)
Comment (Andy-ABTec) [reply _manual/features/finding.md]: I'll go fix it now...

Comment (Andy-ABTec) [reply _manual/features/finding.md]: Not fixed - need to tweak some more,,.

File: _manual/features/finding.md (hunk 1 above)
Comment (Andy-ABTec) [reply _manual/features/finding.md]: Not Fixed - need to tweak some more...

Comment (Andy-ABTec) [reply _manual/features/finding.md]: Fixed (I think) please check...

**Pull request merged**